javav2/usecases/creating_fsa_app/src/main/java/com/example/fsa/handlers/PollyHandler.java

```diff
@@ -0,0 +1,94 @@
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+package com.example.fsa.handlers;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.example.fsa.services.PollyService;
+import com.example.fsa.services.S3Service;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.Map;
+
+public class PollyHandler implements RequestHandler<Map<String, Object>, String> {
+    @Override
+    public String handleRequest(Map<String, Object> requestObject, Context context) {
+        S3Service s3Service = new S3Service();
+        PollyService pollyService = new PollyService();
+        String myValues = requestObject.toString();
+        context.getLogger().log("*** ALL values: " +myValues);
+        String translatedText = getTranslatedText(myValues);
+        String key = getKeyName(myValues);
+        context.getLogger().log("*** About to get bucket");
+        String bucket = getBucketName(myValues);
+        context.getLogger().log("*** My Bucket: " +bucket);
+        String newFileName = convertFileEx(key);
+        context.getLogger().log("*** Translated Text: " +translatedText +" and new key is "+newFileName);
+        try {
+            InputStream is = pollyService.synthesize(translatedText);
+            String audioFile = s3Service.putAudio(is, bucket, newFileName);
+            context.getLogger().log("You have successfully added the " +audioFile +"  in the S3 bucket");
+            return audioFile ;
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    // This method extracts the value following translated_text using Reg Exps.
+    private String getTranslatedText(String myString) {
+        // Define the regular expression pattern to match the "translated_text" key-value pair.
+        String pattern = "translated_text=([^,}]+)";
```

Comment:
Why not just use [Map.get](https://docs.oracle.com/en/java/javase/20/docs/api/java.base/java/util/Map.html#get(java.lang.Object))?